vizro-core/docs/pages/user-guides/dashboard.md

```diff
@@ -201,6 +202,14 @@ Vizro will [automatically incorporate the dashboard logo](assets.md/#add-a-logo-
 
 ![Dashboard with logo](../../assets/user_guides/dashboard/dashboard_with_logo.png)
 
+## Add a dashboard info-icon
+
+The `description` argument enables you to add helpful context to your dashboard by displaying an info icon next to the dashboard title. Hovering over the icon shows a tooltip with your provided text.
+
+You can provide a string to use the default info icon, or pass a custom [`Tooltip`][vizro.models.Tooltip] model to define both a custom icon from the [Google Material Icons library](https://fonts.google.com/icons) and the description text.
```

Comment:
Please can you make the same edit that I made above here? (Or similar change if you decide to modify what I included so it's consistent.